continue-reading.html

```diff
@@ -1,13 +1,155 @@
 <!DOCTYPE HTML>
+<link href="continuereading.css" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=Peddana&display=swap" rel="stylesheet">
+
 <html>
-  <head>
-    <title>Continue Reading</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+  <!--Content Start and Title...-->
+  <a href="index.html" class="continuetitle"> Let's Read More About This!</a>
+  <h2>I guess If You Really Want Too...</h2>
+  <hr class="linebrk">
+  <hr class="linebrk1">

```

Comment:
There are ways to add a double line break with just one element on the page. You can look into using `border: ...` and the different types of things you can do with that